gateleen-security/src/main/java/org/swisspush/gateleen/security/authorization/Authorizer.java

```diff
@@ -119,20 +110,24 @@ public void authorize(final HttpServerRequest request, final Handler<Void> handl
 
         handleUserUriRequest(request, future);
 
-        if(!future.isComplete()){
-            handleIsAuthorized(request, future);
+        if (!future.isComplete()) {
+            roleAuthorizer.handleIsAuthorized(request, future);
+        }
+
+        if (!future.isComplete()) {
+            handleConfigurationUriRequest(request, future, aclUriPattern, roleAuthorizer);
```

Comment:
`aclUriPattern` and `roleAuthorizer` are instance-variables which you pass to an instance-method.
But the method can access the instance-variables by itself - or do I miss something?

Reply:
The method is called with different Objects and patterns.  Don't think this is a problem here.